public/js/include/bootstrap.min.js

```diff
@@ -0,0 +1,7 @@
+/*!
```

Comment:
This is fine for now but in case we're expanding this in the future, we should include bootstrap/js libraries via package.json and browserify for efficiency. This is not a priority for now so let's just make this a TODO when this project is done